README.md

```diff
@@ -44,46 +44,23 @@ hosts and containers in the deployment using the related plays mentioned
 above.
 * `site.yml` - deploys all the above playbooks.
 
-Basic Setup:
-
-1. Clone 
-[os-ansible-deployment](https://github.com/stackforge/os-ansible-deployment).
-2. Clone [rpc-extras](https://github.com/rcbops/rpc-extras).
-3. Prepare the os-ansible-deployment configuration. If you're building an AIO
-you can simply execute `scripts/bootstrap-aio.sh` from the root of the
-os-ansible-deployment clone.
-4. From the root of the os-ansible-deployment clone, execute
-`scripts/bootstrap-ansible.sh`.
-5. Recursively copy `rpc-extras/etc/openstack_deploy/*` to
-`/etc/openstack_deploy/`.
-6. Set the `rpc_repo_path` in
-`/etc/openstack_deploy/user_extras_variables.yml` to the path of the
-`os-ansible-deployment` repository clone directory.
-7. Set all other variables in
-`/etc/openstack_deploy/user_extras_variables.yml` appropriately.
-8. Edit `rpc-extras/playbooks/ansible.cfg` and ensure the paths to the roles,
-playbooks and inventory are correct.
-9. Generate the random passwords for the extras by executing
-`scripts/pw-token-gen.py --file
-/etc/openstack_deploy/user_extras_secrets.yml` from the
-`os-ansible-deployment` clone directory.
-10. Change to the `os-ansible-deployment/playbooks` directory and execute the
-plays. You can optionally execute `scripts/run-playbooks.sh` from the root of
-os-ansible-deployment clone.
-11. Change to the `rpc-extras/playbooks` directory and execute your
-desired plays.  EG:
-
-```bash
-openstack-ansible site.yml
-```
-
-# Ansible Roles
-
-* `beaver`
-* `elasticsearch`
-* `horizon_extensions`
-* `kibana`
-* `logstash`
-* `rpc_maas`
-* `rpc_support`
+# Basic Setup:
 
+1. From `/opt`, clone [rpc-extras](https://github.com/rcbops/rpc-extras)
+with the `--recursive` option to get all the submodules.
+2. Unless doing an AIO build, prepare the os-ansible-deployment configuration.
+  1. copy everything from `os-ansible-deployment/etc/openstack_deploy` 
+     into `/etc/openstack_deploy`
+  2. copy everything from `rpcd/etc/openstack_deploy` into
+     `/etc/openstack_deploy`
+  3. Edit configurations in `/etc/openstack_deploy` for example:
+    1. `openstack_user_variables.yml.example` and
+       `openstack_user_variables.yml.aio`
+    2. There is a tool to generate the inventory for RAX datacenters, otherwise
+       it will need to be coded by hand.
+3. Run `scripts/deploy.sh` within `/opt/rpc-extras`
+  1. If building AIO, set `RPCD_AIO=yes` before running
+  2. If building with Logstash, set `RPCD_LOGSTASH=yes` before running
```

Comment:
I think this var should be renamed RCPD_LOGGING and the word Logstash replaced with something like logging stack to indicate that there are multiple packages installed, containers created and roles ran.